wasm/bindings/response_bindings.cpp

```diff
@@ -10,17 +10,38 @@
 #include "response.h"
 
 using Response = marian::bergamot::Response;
-using SentenceQualityScore = marian::bergamot::Response::SentenceQualityScore;
 using ByteRange = marian::bergamot::ByteRange;
 
+/// Same type as Response::SentenceQualityScore, except with wordByteRanges
+/// instead of wordRanges.
+struct SentenceQualityScore {
+  /// Quality score of each translated word
+  std::vector<float> wordScores;
+  /// Position of each word in the translated text
+  std::vector<ByteRange> wordByteRanges;
+  /// Whole sentence quality score (it is composed by the mean of its words)
+  float sentenceScore = 0.0;
+};
+
 using namespace emscripten;
 
 // Binding code
 EMSCRIPTEN_BINDINGS(byte_range) {
   value_object<ByteRange>("ByteRange").field("begin", &ByteRange::begin).field("end", &ByteRange::end);
 }
 
-std::vector<SentenceQualityScore> getQualityScores(const Response& response) { return response.qualityScores; }
+std::vector<SentenceQualityScore> getQualityScores(const Response& response) {
+  std::vector<SentenceQualityScore> scores;
+  scores.reserve(response.qualityScores.size());
+
+  for (size_t sentenceIdx = 0; sentenceIdx < response.qualityScores.size(); ++sentenceIdx) {
+    scores.emplace_back(SentenceQualityScore{response.qualityScores[sentenceIdx].wordScores,
+                                             marian::bergamot::getWordByteRanges(response, sentenceIdx),
+                                             response.qualityScores[sentenceIdx].sentenceScore});
+  }
+
+  return scores;
+}
```

Comment:
All features expected to be happening at the extension by using `ByteRange` instances (words, sentences) on plain-text translations are now using HTML as the delivery mechanism in proof of concepts:

 * https://github.com/jelmervdl/firefox-translations/pull/11 
 * https://github.com/jelmervdl/firefox-translations/issues/5 
 
 Providing an alternate means of access incurs a maintenance overhead and is impractical to be used at the extension due to part tags being present in these `ByteRange`s potentially.

Reply:
If we ever need these in WASM, we should expose them through the same interface as the other APIs use. For now I've removed these functions.